src/llama-chat.cpp

```diff
@@ -213,6 +214,8 @@ llm_chat_template llm_chat_detect_template(const std::string & tmpl) {
         return LLM_CHAT_TEMPLATE_SEED_OSS;
     } else if (tmpl_contains("'Assistant: '  + message['content'] + '<|separator|>")) {
         return LLM_CHAT_TEMPLATE_GROK_2;
+    } else if (tmpl_contains("[unused9]") && tmpl_contains("message['content'] + '[unused10]'")) {
```

Comment:
```suggestion
    } else if (tmpl_contains("[unused9]") && tmpl_contains("message[\"content\"] + \"[unused10]\"")) {
```
The chat template uses `"`, see
https://huggingface.co/FreedomIntelligence/openPangu-Embedded-7B-V1.1?chat_template=default&format=true

Reply:
> The chat template uses `"`, see https://huggingface.co/FreedomIntelligence/openPangu-Embedded-7B-V1.1?chat_template=default&format=true

Thanks for the suggestion! @CISC 
In tokenizer_config.json, the chat template actually uses single quotes `'`.
In my tests, the condition with `"` didn’t match.
The `"` shown on Hugging Face Hub are likely due to JSON parsing when rendering I guess.

Reply:
<img width="1560" height="441" alt="image" src="https://github.com/user-attachments/assets/a8963760-71e0-461d-9789-1cf273517be1" />

Reply:
I’ve addressed other comments and updated the PR. Thanks again for your review!

Reply:
Ah, wasn't aware HF changed that when pretty-printing, will watch out for that in the future!

I was hoping to avoid the unicode matching, but I guess it's safer.